codalib/util.py

```diff
@@ -11,8 +11,7 @@
 from lxml import etree
 import uuid
 import datetime
-from coda import bagatom
-import httplib2
```

Comment:
If we are taking out this import, let's delete this commented out use:
https://github.com/unt-libraries/codalib/blob/add-setup/codalib/util.py#L65